dotCMS/src/main/java/com/dotmarketing/quartz/job/CleanUpFieldReferencesJob.java

```diff
@@ -101,34 +113,26 @@ public void run(JobExecutionContext jobContext) throws JobExecutionException {
             Logger.error(CleanUpFieldReferencesJob.class,
                     "Error cleaning up field references. Field velocity var: " + field.variable(), e);
         }
+*/
+
+        try {
+            Thread.sleep(25000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        System.out.println("::: done!");
```

Comment:
Codacy found an issue: [System.out.println is used](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6077927)